src/main.js

```diff
@@ -16,24 +28,32 @@ const render = (container, template, place) => {
 const siteBodyElement = document.querySelector(`.page-body`);
 const tripMainElement = siteBodyElement.querySelector(`.trip-main`);
 
-render(tripMainElement, createTripInfoTemplate(), `afterbegin`);
+render(tripMainElement, createTripInfoTemplate(waypoints), `afterbegin`);
 
 const siteControlsElement = tripMainElement.querySelector(`.trip-controls`);
 
-render(siteControlsElement, createSiteMenuTemplate(), `afterbegin`);
-render(siteControlsElement, createTripFiltersTemplate(), `beforeend`);
+render(siteControlsElement, createSiteMenuTemplate(menuItems), `afterbegin`);
+render(siteControlsElement, createTripFiltersTemplate(filters), `beforeend`);
 
 const tripEventsElement = siteBodyElement.querySelector(`.trip-events`);
 
-render(tripEventsElement, createTripSortingTemplate(), `beforeend`);
-render(tripEventsElement, createTripListTemplate(), `beforeend`);
+render(tripEventsElement, createTripSortingTemplate(sortingItems), `beforeend`);
+render(tripEventsElement, createTripTemplate(waypoints), `beforeend`);
```

Comment:
`createTripTemplate` не использует никакие данные.
```suggestion
render(tripEventsElement, createTripTemplate(), `beforeend`);
```

Reply:
Все верно, поправила у себя.